src/Containers/ClientShowCase.tsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import styled from 'styled-components';
+import { Button } from '../Inputs/Button';
+
+const IMAGE_HEIGHT = 30;
+
+export interface ClientProps extends React.HTMLAttributes<HTMLDivElement> {
+    imgData: string[];
+    handleClick: (event: React.MouseEvent<Element, MouseEvent>) => void;
+    buttonText: string;
```

Comment:
Rather than button text, perhaps just allow the user to pass in a button if they want one